### PR TITLE
Add link to new actions

### PR DESCRIPTION
**Pull request by kissmikijr**
## Hey, I just made a Pull Request!
This PR adds links to some new scaffolder actions that we created

#### :heavy_check_mark: Checklist

- [ ] A changeset describing the change and affected packages. ([more info](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#creating-changesets))
- [ ] Added or updated documentation
- [ ] Tests for new functionality and regression tests for bug fixes
- [ ] Screenshots attached (for UI changes)
- [x] All your commits have a `Signed-off-by` line in the message. ([more info](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#developer-certificate-of-origin))

### CODE REVIEW

File: microsite/data/plugins/scaffolder-backend-roadie-utils.yaml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+---
+title: Scaffolder utils actions
```
Comment (jhaals) [inline microsite/data/plugins/scaffolder-backend-roadie-utils.yaml:2]:
```suggestion
title: Scaffolder utility actions
```

File: microsite/data/plugins/scaffolder-backend-roadie-utils.yaml [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+author: roadie.io
+authorUrl: https://roadie.io/?utm_source=backstage.io&utm_medium=marketplace&utm_campaign=scaffolder-utils
+category: Scaffolder
+description: A collection of actions that contains some utility functions
```
Comment (jhaals) [inline microsite/data/plugins/scaffolder-backend-roadie-utils.yaml:6]:
```suggestion
description: A collection of utility actions including sleep, zip and file manipulation.
```

**Review by jhaals — COMMENTED** [review]
Thanks! Some suggestions for the title and description for the utility module

File: microsite/data/plugins/scaffolder-backend-roadie-utils.yaml (hunk 1 above)
Comment (kissmikijr) [reply microsite/data/plugins/scaffolder-backend-roadie-utils.yaml]: Thanks! Changed it!

File: microsite/data/plugins/scaffolder-backend-roadie-aws.yaml [hunk 3]
```diff
@@ -0,0 +1,9 @@
+---
+title: Scaffolder AWS cli actions
+author: roadie.io
+authorUrl: https://roadie.io/?utm_source=backstage.io&utm_medium=marketplace&utm_campaign=scaffolder-aws
+category: Scaffolder
+description: Here you can find some AWS cli actions
+documentation: https://github.com/RoadieHQ/roadie-backstage-plugins/blob/main/plugins/scaffolder-actions/scaffolder-backend-module-aws/README.md
+iconUrl: img/scaffolder-aws.-logo.png
```
Comment (Rugvip) [inline microsite/data/plugins/scaffolder-backend-roadie-aws.yaml:8]: Could you perhaps find a public source for this logo instead? We prefer to not host logos within the repo

**Review by Rugvip — COMMENTED** [review]
Thank you! Just a minor nit

**Review by jhaals — APPROVED** [review]
Thank you! 👍

**Pull request merged**